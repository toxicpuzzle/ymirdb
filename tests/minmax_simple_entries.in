SET a 1 2 3 4 5
SET b 20 -10 2
MIN a
MAX a
MIN b
MAX b
bye