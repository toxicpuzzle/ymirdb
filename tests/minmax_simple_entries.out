> ok

> ok

> 1

> 5

> -10

> 20

> bye
